.travis.yml

```diff
@@ -2,5 +2,6 @@ dist: xenial
 language: python
 python: 3.7
 cache: pip
-before_install: pip install -U pre-commit
+before_install:
```

Comment:
Plz split commits to have testing-related changes separately. Atomic changes are preferrable.